src/pages/UpdateServers.js

```diff
@@ -52,14 +54,15 @@ class UpdateServers extends BaseUpdateWizardPage {
 
       showSharedWarning: false,
 
-      serverToReplace: undefined,
```

Comment:
FYI, a trailing comma in object lists is officially supported in JavaScript now, so you don't have to remove these, but I won't object if you do.